cmake/modules/DefineInstallVar.cmake

```diff
@@ -13,12 +13,12 @@ function(DEFINE_INSTALL_VAR)
             "${multiValueArgs}" ${ARGN} )
 
     # install(CODE) does not support generator expression in ver<3.14
```

Comment:
On a side note, `CMP0087` should be set to use generator expressions in `CMake >= 3.14`.